pagerduty/plugin.spec.yaml

```
@@ -69,6 +69,15 @@ types:
       - read_only_user
       - user
 actions:
+  get_on_call:
+    title: Get List of Users on Call
+    description: Get List of Users on Call
```

```suggestion
    description: Get list of on-call users
```
Should conform to the style guide (sentence case): https://komand.github.io/python/spec.html#description